README.md

```diff
@@ -32,77 +32,83 @@ Quandl::ApiConfig.api_version = '2015-04-09'
 
 ## Retrieving Data
 
-### Database
+### Dataset
 
-To retrieve a database simply use its code with the get parameter:
+Retrieving dataset data can be done in a similar way to Databases. For example to retrieve a dataset use its full code: 
 
 ```ruby
 require 'quandl'
-Quandl::Database.get('WIKI')
-=> ... wiki database ...
+Quandl::Dataset.get('WIKI/AAPL')
+=> ... dataset ...
 ```
 
-You can also retrieve a list of databases by using: 
+You can also retrieve a list of datasets associated to a database by using the Database helper method:
+
 
 ```ruby
-require 'quandl'
-Quandl::Database.all
-=> ... results ...
+Quandl::Database.get('WIKI').datasets
+=> ... datasets results ...
 ```
 
-You can also search for specific databases by passing a query parameter such as:
+By default, each list query will return page 1 with the first 100 results (Please see the [API Documentation](https://www.quandl.com/docs/api) for more detail)
```

Comment:
Maybe a slight re-wording: `By default, each list query will return page 1 of the first 100 results...`